README.md

```diff
@@ -507,6 +525,44 @@ function returnsImpossible() {
 // type of returnsImpossible() is 'never'
 ```
 
+## Difference types
+
+### Flow
+
+```js
+type C = $Diff<{ a: string, b: number }, { a: string }>
+// C is { b: number}
+```
+
+Note however that $Diff is not an official feature. 
+
+> It only works properly as lower bound, i.e. you can assign something to it, but can't use it after that.
+
+([source](https://github.com/facebook/flow/issues/3541#issuecomment-289291932)]
+
+### Typescript
+
+You can define your own filter using conditional typing, but it does not have a build type mapper for that.
```

Comment:
Could you elaborate on what a `build type mapper` is? I've googled the term to no avail 🧐

Reply:
I mean built-in. Maybe change this to `it does not have a helper type for that`. It's just that Typescript refers to `helper types` as `type mappers`.